src/main/java/hu/bme/mit/spaceship/GT4500.java

```diff
@@ -74,14 +73,11 @@ public boolean fireTorpedo(FiringMode firingMode) {
             // if both of the stores are empty, nothing can be done, return failure
           }
         }
-        break;
-
-      case ALL:
+    }
+    else if(firingMode == firingMode.All){
         // try to fire both of the torpedo stores
 
 	firingSuccess = true;
```

Comment:
WTF is this, alwasy true???

Reply:
Oh, thanks for asking, i forgot this. I'll fix it.